skopt/optimizer/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .dummy_opt import dummy_minimize
+from .forest_opt import forest_minimize
+from .gbrt_opt import gbrt_minimize
+from .gp_opt import gp_minimize
```

Comment:
Since those now live in the `optimizer` module, shouldnt we drop the `_opt` suffix?